docs/architecture/api.mdx

```diff
@@ -0,0 +1,526 @@
+---
+id: api
+title: Hotspot API Documentation
+pagination_label: Hotspot API
+sidebar_label: Hotspot API Documentation
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /architecture/hotspot-api
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Hotspot API Documentation
+
```

Comment:
we should add a blurb in here that describes the source of the data, who operates it, etc.